android/build.gradle

```diff
@@ -29,6 +29,7 @@ def getExtOrIntegerDefault(name) {
 apply plugin: 'com.android.library'
 
 android {
+  namespace = "com.reactnativecommunity.netinfo"
```

Comment:
Random comment, you could have also defined something like this:

```groovy
def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
// Check AGP version for backward compatibility reasons
if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
   namespace = "com.reactnativecommunity.netinfo"
}
```

Reply:
PRs happily accepted :-)

Reply:
sure, one moment :)

Reply:
on second thought, adding the namespace to build.gradle doesn't create any problems for people that don't care about it (so conditionally adding it is...not interesting?) - it's the inability to remove it from AndroidManifest without triggering a breaking change that's the issue, isn't it?

Reply:
I understood that this would cause build issues on Gradle 6 or older. I have seen a number of packages adopting the above change instead of just adding "namespace" to the gradle file.